src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -59,7 +59,8 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
         });
 
         mapControl = new MapControl({
-            tabControl: tabControl
+            tabControl: tabControl,
+            isMobile: $(window).width() < MD_UP_BREAKPOINT
```

Comment:
Since this comparison is being done twice now, it could be moved to a component-level variable set during initialization.  Then again, it's only twice.